plans/provision_compiler.pp

```diff
@@ -23,9 +24,16 @@
         action => 'get',
         setting => 'certname'
       ).find($target.name).value['status']
-
+  
+    if $legacy_compiler == true {
```

Comment:
There is an indenting issue here. I would expect these lines to be two spaces further indented.